fla/modules/convolution.py

```
@@ -12,6 +12,7 @@
 
 from fla.ops.utils import prepare_chunk_indices, prepare_sequence_ids
 from fla.utils import IS_AMD, autotune_cache_kwargs, get_multiprocessor_count, input_guard
+from fla.ops.common.cp_chunk_delta_h import get_gdn_cp_context, pre_process_for_conv1d, post_process_for_conv1d
```

Flake8 (7.3.0)

[error] 'fla.ops.common.cp_chunk_delta_h.get_gdn_cp_context' imported but unused

(F401)